pyspeckit/spectrum/fitters.py

```diff
@@ -1448,7 +1416,7 @@ def moments(self, fittype=None, **kwargs):
             The registered fit type to use for moment computation
         """
         fittype = fittype or self.fittype
-        return self.Registry.singlefitters[fittype].moments(
+        return self.Registry.multifitters[fittype].moments(
```

Comment:
This relates to the comment above: https://github.com/pyspeckit/pyspeckit/pull/25/files#r26297611.  We should keep `singlefitters` around and allow them to be used in this function.  BUT, most of the rest of your changes are right: we want to remove the `multifit` option. 

Maybe it would also make sense to rename `singlefitters` to `peakbgfitters`, since that's what they'll be used for from now on.